content/docs/user-guide/networking/multiplayer/overview.md

```diff
@@ -45,6 +45,30 @@ Then, clients update themselves from a delta by:
 
 After, the AzNetworking layer becomes aware that the world was updated. This occurs automatically through ACK vector replication, and does not require any further action from the client.
 
+### Types of network properties
+
+Network properties have two important fields: `ReplicateFrom` and `ReplicateTo`. 
+Together, these define which role can replicate to which role. You can only replicate property values *from* Authority and Autonomous roles. Under the server-authority model of O3DE, this ensures that you never accidentally replicate from an unprivileged client. 
+Properties can be replicated *to* any role, since all active clients in the session must share information and some may not necessarily need to be used in the server-authoritative model.
```

Comment:
Should it maybe be changed to `all participants in the session`?

```suggestion
Properties can be replicated *to* any role, since all participants in the session may need information from any other participant.
```